### DISCUSSION TITLE
Forgot keplr wallet app password. Got bk in w/seed. but now my staked AKT is not in there.

### DISCUSSION THREAD

**Discussion by BTCBanditXIII** [akash-network/community#704] [Q&A]
new to akt, But not to crypto. 

Got aquainted to Akash/AKT/Keplr app. thru some quests on coinbase. 
Once I finished the Akash/ AKT quests, I had staked the chump change coinbase gave me with a validator of my choice.

Couple days later I performed some updates to my phone next time I go to sign in I noticed my saved password for the keplr app wasn't filling in on the sign in screen. 

Long story short  somehow some of my passwords to a few apps disappeared not sure how... Anyways 

Needless to say, I guess i can say I Forgot keplr wallet app password. 
So I couldn't get in for a day or so. 
Found my seed phrase, Used it to gain access into the keplr wallet app after having to reinstall it. 

After reinstalling keplr app i imported my wallets seed phrase, but the 2 wallets I had in the wallet app that were both holding AKT ON THE AKASH NETWORK are no longer in there. But I have my wallet addresses both the one holding akt, & the one that's got staked akt on the akash network. 
![Screenshot_20241012-021834_Keplr](https://github.com/user-attachments/assets/503dde03-8a5d-4898-9e3d-7022f467e2c4)
![Screenshot_20241012-021804_Keplr](https://github.com/user-attachments/assets/15273add-e8a0-4056-ab35-f97e5de3569c)
![Screenshot_20241020-111059_Keplr](https://github.com/user-attachments/assets/cb5f8043-c709-4691-88b1-55130c055332)

**Comment by rodri-r** [comment]
Hey in your second screenshot, you can see that you have 3.24 AKT staked

**Reply by BTCBanditXIII** [reply]
Yes that was the akt balance that's disappeared from the same wallet I forgot password to, that has also since been restored. But as I mentioned in the original post, once I re-installed the wallet app... & Got back into my wallet, there was suppose to be 2 separate AKT wallets already set up from the coinbase quests, but once I restored the wallet from my seedphrase nothing was in it. Neither 1 of my alt wallets are in it.

**Comment by nelsonizy4-beep** [comment]
There are a lot of untrue recommendations and its hard to tell who is legit. If you have lost money or crypto coin to scam contact (hack4net8@ gmail .com) they will surely help you out.
